content/specs/repository.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Repository
+summary: Self-authenticating storage for public account content
+---
+
+# Repository
+
+*See the [Data Repositories Guide](../guides/data-repos) document for a higher-level introduction.*
+
+Public atproto content ("records") is stored in per-account "repositories" (frequently shortened to "repo"). All currently active records are stored in the repository, and current repository contents are publicly available, but both content deletions and account deletions are fully supported.
+
+The repository data structure is content-addressed (a Merkle-tree), and every mutation of repository contents (eg, addition, removal, and updates to records) results in a new "commit" version. Commits are cryptographically signed, with rotatable "signing keys", which allows recursive validation of content as a whole or in part.
+
+Repositories and their contents are canonically stored in binary DAG-CBOR format, as a graph of IPLD data objects referencing each other by content hash (CID Links). Large binary "blobs" are not stored directly in repositories, though they are referenced by hash (CID). This includes images are other media objects. Repositories can be exported as CAR files for offline backup, account migration, or other purposes.
+
+In the atproto federation architecture, the authoritative location of an account's repository is the associated Personal Data Server (PDS). An account's current PDS location is authoritatively indicated in the DID Document.
+
+In real-world use, it is expected that individual repositories will contain anywhere from dozens to millions of records.
+
+
+## Repo Data Structure (v2)
+
+This describes version `2` of the repository binary format. Version `1` had a different MST fanout configuration, and an incompatible schema for commits and repository metadata.
```

Comment:
maybe a quick note that v1 is completely deprecated & support is not required by any service